tuples.go

```diff
@@ -258,7 +260,19 @@ func (c *Client) DeleteAllObjectRelations(ctx context.Context, object string) er
 	}
 
 	// Notes: Writes only allow 10 tuples per call, this will need to be fixed
-	_, err = c.DeleteRelationshipTuple(ctx, tuplesToDelete)
+	// _, err = c.DeleteRelationshipTuple(ctx, tuplesToDelete)
+
+	for i := 0; i < len(tuplesToDelete); i += defaultSize {
+		end := i + defaultSize
+		if end > len(tuplesToDelete) {
+			end = len(tuplesToDelete)
+		}
+		allTuples := tuplesToDelete[i:end]
+		_, err := c.DeleteRelationshipTuple(ctx, allTuples)
+		if err != nil {
+			return err
+		}
```

Comment:
this is a personal preference, but a short line like this with an err check can be rewritten like:

```suggestion
		if _, err := c.DeleteRelationshipTuple(ctx, allTuples); err != nil {
			return err
		}
```